test/classes/corrado/type_alias_bug.verify.skipif

```diff
@@ -0,0 +1 @@
+CHPL_NIGHTLY_TEST_CONFIG_NAME != verify
```

Comment:
Alternatively, and my preference, is instead of .skipif to have a .compopts that contains `--verify`